etc/kayobe/kolla/config/monasca/persister.conf

```diff
@@ -1,2 +1,12 @@
 [kafka_alarm_history]
 enabled = False
+
+# This prevents the persister jamming on bad metrics. Metrics which are
+# skipped are logged in the persister log file.
+[repositories]
+ignore_parse_point_error = True
```

Comment:
I'm not sure why you would ever want to jam on bad metrics :)

Reply:
I suppose it forces you to fix the issue quickly. They always come from log-metrics which really needs a Logstash plugin to post them via the Monasca API, rather than direct to Kafka.